libcaf_core/caf/chrono.cpp

```diff
@@ -151,7 +151,7 @@ char* print_fractional_component(char* pos, int ns, int precision,
   return pos;
 }
 
-char* print_utf_offset(char* pos, int offset) {
+char* print_utf_offset(char* pos, long int offset) {
```

Comment:
```suggestion
char* print_utf_offset(char* pos, long offset) {
```

Writing `long int` is a waste of space. No reason to type it, just like no one is actually writing `signed int`. 😉